components/BlogFeed.js

```diff
@@ -0,0 +1,164 @@
+/* eslint-disable react/no-danger */
+
+import React from 'react';
+import Truncate from 'react-truncate';
+import styled from 'styled-components';
+import MarkdownIt from 'markdown-it';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import Link from 'next/link';
+import CloudinaryImage from './CloudinaryImage';
+import Parser from './Parser';
+import theme from '../theme';
+
+export default function BlogFeed({ posts }) {
+  const md = new MarkdownIt();
+  if (posts === 'undefined') return null;
+
+  return (
+    <StyledBlogFeed>
+      <div className="blog blog__container">
+        <h2 className="blog__heading">TJI Blog</h2>
+        <span className="blog__tagline">
+          Where the TJI team posts blogs about our data, publications, analyses, and code.
+        </span>
```

Comment:
Could we use a header that looks more like the headers on some of our other pages? e.g. https://texasjusticeinitiative.org/publications